modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -0,0 +1,54 @@
+= Troubleshooting API Provisioning Issues
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, API provisioning issues, troubleshooting
+
+After you provision an adapter, you must create an API, to which you then bind the service. You might encounter errors when you provision an API if you provide invalid credentials, and so on. 
+
+//Should I move this error to the Service errors topic and call the topic "API and Service Issues"?
+
+== Error With Parameters
```

Comment:
```suggestion
== Credential Errors
```

Reply:
But, again, there is only one problem in this section. Do you anticipate more in future?

Reply:
Made a comment to ask Dev.